app/controllers/users/passwords_controller.rb

```diff
@@ -84,6 +86,16 @@ def handle_unsuccessful_password_reset
       render :edit
     end
 
+    def mark_profile_inactive
+      active_profile = resource.active_profile
+      return unless active_profile.present?
+      active_profile.update!(active: false)
+      analytics.track_event(
```

Comment:
Does it make sense to track this *and* password reset if they both happen together?